t/api/13-influxdb.t

```diff
@@ -57,16 +57,19 @@ $t->get_ok('/admin/influxdb/minion')->status_is(200)
   ->content_like(qr!openqa_minion_workers,url=http://example.com active=0i,inactive=1i,registered=1i!);
 
 my $dbh = $schema->storage->dbh;
-my $rc_fail_finished = DateTime->now->truncate(to => 'minute');
-$rc_fail_finished->subtract(minutes => ($rc_fail_finished->minute() % 10) + 3);
+my $rc_fail_timespan_end = DateTime->now->truncate(to => 'minute');
+$rc_fail_timespan_end->subtract(minutes => ($rc_fail_timespan_end->minute() % 10));
+my $rc_fail_timespan_end_ns = $rc_fail_timespan_end->epoch() * 1e9;
+my $rc_fail_finished = $rc_fail_timespan_end->clone();
+$rc_fail_finished->subtract(minutes => 3);
 my $sth = $dbh->prepare(
 q!INSERT INTO minion_jobs (id, args, created, delayed, finished, priority, result, retried, retries, started, state, task, worker, queue, attempts, parents, notes)
 	VALUES (7291599, '["/bin/true", 11201356, {"delay": 60, "retries": 1440, "skip_rc": 142, "timeout": "10m", "kill_timeout": "10s"}]', '2023-05-26 17:00:50.542916+02', '2023-05-26 17:00:50.542916+02', ?, 0, null, null, 0, '2023-05-26 17:00:50.565839+02', 'finished', 'hook_script', 1388, 'default', 1, '{}', '{"hook_rc": -1, "hook_cmd": "foobar", "hook_result": "Job is '':investigate:'' already, skipping investigation\n"}')!
 );
 $sth->execute($rc_fail_finished);
 $t->get_ok('/admin/influxdb/minion')->status_is(200)
   ->content_like(qr!openqa_minion_jobs,url=http://example.com active=0i,delayed=1i,failed=0i,inactive=2i!)
-  ->content_like(qr!openqa_minion_jobs_hook_rc_failed,url=http://example.com rc_failed_per_10min=1i \d+!)
+  ->content_like(qr!openqa_minion_jobs_hook_rc_failed,url=http://example.com rc_failed_per_10min=1i $rc_fail_timespan_end_ns!)
```

Comment:
I think would actually just check for the right amount of digits (19). That should work for quite a while :)